gsa/src/web/pages/alerts/method.js

```diff
@@ -59,13 +62,26 @@ const Table = styled(SimpleTable)`
   }
 `;
 
-const Method = ({method = {}, details = false}) => {
+const Pre = styled.pre`
+  white-space: pre-wrap;
+  word-wrap: break-word;
+`;
+
+const Method = ({method = {}, details = false, reportFormats}) => {
   if (!isDefined(method.type)) {
     return null;
   }
+
+  const getReportFormatName = id => {
+    if (isDefined(reportFormats)) {
```

Comment:
This if statement can be dropped if reportFormats is initialized as an empty array as my suggestion above does.